src/legacy/core_plugins/vis_type_timelion/public/components/_index.scss

```diff
@@ -1 +1,2 @@
+@import './chart';
```

Comment:
The `_chart.scss` file should be renamed to `_panel.scss` to match it's corresponding JS file